### DISCUSSION TITLE
Selection an area

### DISCUSSION THREAD

**Discussion by dominiosc** [sumatrapdfreader/sumatrapdf#2714] [Q&A] [answered]
I remember that in previous versions you could select an area and copy as an image, have you missed this feature?

**Comment by dominiosc** [comment, marked as answer]
I remember, use the control key + mouse

Is there a way to select the active page?
https://sumatrapdf.canny.io/feature-requests/p/select-this-page

**Reply by kjk** [reply]
For things that are images (image files, cbr, PDF files where a page is just an image) you can double-click to select the while image / page.

**Reply by GitHubRulesOK** [reply]
@dominiosc 
You need to be in one page mode not all pages (NOT continuous) or select a part area
![image](https://user-images.githubusercontent.com/12754913/173208577-6cf6f9f5-97e4-4f13-9bbc-00efc7cf4817.png)
